e2e/cypress/integration/multi_team_and_dm/gm_add_first_user_spec.js

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+/* eslint-disable jquery/no-text */
+/* eslint-disable jquery/no-find */
+
+describe('Multi-user group messages', () => {
+    const userPrefix1 = '0aadam';
+    const userPrefix2 = '0aabadam';
+    const userPrefix3 = 'beatrice';
+    const searchTerm = '0aa';
```

Comment:
In the CI, what dataset is originally present in the env before running the cypress tests. Is it the same data set as `make test-data` from `mattermost-server` ? Just wondering how this can affect the display order of the users that I am creating here.

Reply:
Before running the entire, no data is present in the DB. We're creating all the test data as needed by the test (including sysadmin user), and gets accumulated as we run each test.